{{cookiecutter.project_slug}}/docker-compose.yml

```diff
@@ -27,6 +30,25 @@ services:
       - ~/.cache/pip:/home/nens/.cache/pip  # shared pip cache
       - ~/.cache/pipenv:/home/nens/.cache/pipenv  # shared pipenv cache
 
+{% if cookiecutter.celery == "yes" %}  celery:
```

Comment:
Bij deze en bij een paar andere plekken: ik zou gewoon een 'enter' doen na de if tag. Dan maar een extra witregel in de output. Ik vind de indentatie van de eerste regel (omdat het achter de if staat) wat lastig leesbaar.

(Mits het qua syntax uiteraard kan).

Reply:
waar is die `C_FORCE_ROOT` voor nodig?